src/patternfly/components/Sequence/examples/sequence-complex-example.hbs

```diff
@@ -0,0 +1,122 @@
+					{{#> sequence-content-title}}Step One{{/sequence-content-title}}
+					{{#> sequence-content-status}}Status{{/sequence-content-status}}
+				{{/sequence-content}}
+			{{/sequence-link}}
+		{{/sequence-item}}
+		{{#> sequence-item sequence-item--attribute='aria-setsize="10" aria-posinset="2"'}}
+			{{#> sequence-link sequence-link--attribute='href="#"'}}
+				{{#> sequence-number-container}}
+					{{#> sequence-number}}2{{/sequence-number}}
+				{{/sequence-number-container}}
+				{{#> sequence-content}}
+					{{#> sequence-content-title}}Step Two{{/sequence-content-title}}
+					{{#> sequence-content-status}}Status{{/sequence-content-status}}
+				{{/sequence-content}}
+			{{/sequence-link}}
+		{{/sequence-item}}
+	  {{#> sequence-item sequence-item--attribute='aria-setsize="10" aria-posinset="3"'}}
+			{{#> sequence-link sequence-link--attribute='href="#"'}}
+				{{#> sequence-number-container}}
+					{{#> sequence-number}}3{{/sequence-number}}
+				{{/sequence-number-container}}
+				{{#> sequence-content}}
+					{{#> sequence-content-title}}Step Three{{/sequence-content-title}}
+					{{#> sequence-content-status}}Status{{/sequence-content-status}}
+				{{/sequence-content}}
+			{{/sequence-link}}
+		{{/sequence-item}}
+		{{#> sequence-item sequence-item--attribute='aria-setsize="10" aria-posinset="4"'}}
+			{{#> sequence-link sequence-link--attribute='href="#"'}}
+				{{#> sequence-number-container}}
+					{{#> sequence-number}}4{{/sequence-number}}
+				{{/sequence-number-container}}
+				{{#> sequence-content}}
+					{{#> sequence-content-title}}Step Four{{/sequence-content-title}}
+					{{#> sequence-content-status}}Status{{/sequence-content-status}}
+				{{/sequence-content}}
+			{{/sequence-link}}
+		{{/sequence-item}}
+		{{#> sequence-item sequence-item--attribute='aria-setsize="10" aria-posinset="5"'}}
+			{{#> sequence-link sequence-link--attribute='href="#"'}}
+				{{#> sequence-number-container}}
+					{{#> sequence-number}}5{{/sequence-number}}
+				{{/sequence-number-container}}
+				{{#> sequence-content}}
+					{{#> sequence-content-title}}Step Five{{/sequence-content-title}}
+					{{#> sequence-content-status}}Status{{/sequence-content-status}}
+				{{/sequence-content}}
+			{{/sequence-link}}
+		{{/sequence-item}}
+		{{#> sequence-item sequence-item--attribute='aria-setsize="10" aria-posinset="6"'}}
+			{{#> sequence-link sequence-link--attribute='href="#"'}}
+				{{#> sequence-number-container}}
+					{{#> sequence-number}}6{{/sequence-number}}
+				{{/sequence-number-container}}
+				{{#> sequence-content}}
+					{{#> sequence-content-title}}Step Six{{/sequence-content-title}}
+					{{#> sequence-content-status}}Status{{/sequence-content-status}}
+				{{/sequence-content}}
+			{{/sequence-link}}
+		{{/sequence-item}}
+		{{#> sequence-item sequence-item--attribute='aria-setsize="10" aria-posinset="7"'}}
+			{{#> sequence-link sequence-link--attribute='href="#"'}}
+				{{#> sequence-number-container}}
```

Comment:
Just a general comment about `sequence-number-container`. The current/hover states cause the link width to change because the number size changes. IMO, the links shouldn't shift when you hover over or make a link active.

Also, if the sequence number will always be a number, I wonder if we could use a CSS counter and display the number as pseudo elements instead of in the markup. Just an idea.

Reply:
I made a change to reduce the width of the spacer between the number container and content to be the width of the increase in width from the small number div to the large number div. So now it doesnt change on hover. Is this the right approach?

Reply:
> Also, if the sequence number will always be a number, I wonder if we could use a CSS counter and display the number as pseudo elements instead of in the markup. Just an idea.

@michael-coker Is that similar to the examples shown here: https://css-tricks.com/custom-list-number-styling/

Yesterday I had tested these examples in JAWS, plus [variations on lists](https://codepen.io/jenngiardino/pen/jeBreP?editors=1000.) in general. Some general observations:

- `<ol>` list items only announce the list item number if the `list-style` property isn't set to none.
- Navigating the css-tricks examples using JAWS resulted in separate stops for the counter and the list item contents. 

My recommendation would be to aim for having the list item counter and contents announced together, e.g. "link 1 Information". Doing this seems to be dependent on using inline elements, like `<a>` or `<span>`. But using `<div>` resulted in multiple keypresses, so we should change those to `<span>` elements if possible.

Reply:
@jgiardino yep that's it. thanks for testing that! good to know!